customer-service/src/main/java/io/eventuate/examples/tram/ordersandcustomers/customers/service/CustomerService.java

```diff
@@ -50,12 +55,11 @@ public CustomerService(CustomerRepository customerRepository,
             .flatMap(customer ->
                     domainEventPublisher
                             .publish(Customer.class, customer.getId(), customerWithEvents.events)
-                            .collectList()
-                            .thenReturn(customer))
+                            .map(notUsed -> customer))
```

Comment:
`.map(notUsed -> customer)` is `.thenReturn(customer)` 

https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html#thenReturn-V-